src/details/ArborX_DetailsDistributedSearchTreeImpl.hpp

```diff
@@ -502,7 +476,8 @@ void DistributedSearchTreeImpl<DeviceType>::sortResults(
   Kokkos::deep_copy(keys_clone, keys);
   auto const permutation = ArborX::Details::sortObjects(keys_clone);
 
-  applyPermutations(permutation, other_views...);
+  auto dummy = {
+      (ArborX::Details::applyPermutations(permutation, other_views), 0)...};
```

Comment:
This is being too clever.  I am even surprised the compiler does not complain about forming a reference to void in the initializer list.
At the very least you need to comment.

Reply:
This line throws a warning in clang:
```
In file included from /var/jenkins/workspace/ArborX_PR-197/test/tstDetailsDistributedSearchTreeImpl.cpp:14:
/var/jenkins/workspace/ArborX_PR-197/src/details/ArborX_DetailsDistributedSearchTreeImpl.hpp:479:8: warning: unused variable 'dummy' [-Wunused-variable]
  auto dummy = {
       ^
```